.devcontainer/docker-compose.yml

```diff
@@ -16,11 +16,11 @@ services:
   # data portal ui
 
   data_portal_ui_svc:
-    image: ghga/sandbox-ui:dev
+    image: ghga/data_portal_ui_v0:dev
     environment:
-      ghga_data_portal_v0_ui_HOST: 0.0.0.0
-      ghga_data_portal_v0_ui_PORT: 8080
-      ghga_data_portal_v0_ui_WELCOME_INFO: This web app is part of an implementation study.\nIt's not intended for production use.
+      data_portal_ui_v0_HOST: 0.0.0.0
+      data_portal_ui_v0_PORT: 8080
+      data_portal_ui_v0_WELCOME_INFO: This web app is part of an implementation study.\nIt's not intended for production use.
```

Comment:
Is this needed anywhere?

Reply:
Seems not, at least not now. This will be needed for deployment when we have an URL and port we need to use for the data portal, I think.

Reply:
I meant the welcome info

Reply:
Well, I deleted all the variables!